services/meta/errors.go

```diff
@@ -55,6 +55,10 @@ var (
 	// with an existing policy.
 	ErrRetentionPolicyConflict = errors.New("retention policy conflicts with an existing policy")
 
+	// ErrRetentionPolicyDurationBelowShardDuration is returned when updating a retention
+	// policy that has a duration lower than the current shard duration.
+	ErrRetentionPolicyDurationBelowShardDuration = errors.New("retention policy duration must be greater than the shard duration")
```

Comment:
@jsternberg how about changing `ErrRetentionPolicyDurationTooLow` to `ErrRetentionPolicyDurationInvalid`, and then instead of `ErrRetentionPolicyDurationBelowShardDuration` have `ErrRetentionPolicyDurationTooLow` ?